backend/tracim_backend/config.py

```diff
@@ -164,14 +164,18 @@ def load_config(self) -> None:
 
         upload_permissions_config.load_config(self)
 
+    def _here_replace(self, value: str) -> str:
```

Comment:
This can be a public method (you use it outside)

Reply:
done and renamed